demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -329,6 +332,40 @@ def test_should_file_skip_validation_positive(self, file_path):
         assert should_skip
 
 
+class TestGetRemoteFileLocally:
+    REPO_NAME = 'example_repo'
+    FILE_NAME = 'somefile.json'
+    FILE_CONTENT = '{"id": "some_file"}'
+
+    def setup_method(self):
+        # create local git repo
+        example_repo = git.Repo.init(self.REPO_NAME)
+        example_repo.git.checkout('-b', 'origin/master')
+        with open(os.path.join(self.REPO_NAME, self.FILE_NAME), 'w+') as somefile:
+            somefile.write(self.FILE_CONTENT)
+        example_repo.git.add(self.FILE_NAME)
+        example_repo.git.config('user.email', 'automatic@example.com')
+        example_repo.git.config('user.name', 'AutomaticTest')
+        example_repo.git.commit('-m', 'test_commit', '-a')
+        example_repo.git.checkout('-b', 'master')
+
+    def test_get_file_from_master_when_in_private_repo(self, mocker):
+        mocker.patch.object(tools, 'is_external_repository', return_value=True)
+
+        class Response:
+            ok = False
+
+        mocker.patch.object(requests, 'get', return_value=Response)
```

Comment:
next time, you can use ["requests-mock"](https://pypi.org/project/requests-mock/)